src/ScheduleFunctions.cpp

```diff
@@ -1368,20 +1222,20 @@ class InjectGroupRealization : public IRMutator2 {
             const auto &f2_it = env.find(pair.func_2);
             internal_assert(f2_it != env.end());
             const vector<Dim> &dims_2 =
-                (pair.stage_2 == 0) ? f2_it->second.definition().schedule().dims() :
-                                      f2_it->second.update(pair.stage_2-1).schedule().dims();
+                    (pair.stage_2 == 0) ? f2_it->second.definition().schedule().dims() :
```

Comment:
While you're reindenting things, I would prefer the style:
```
 (cond ? 
  true_value :
  false_value)
```

this is in line with Steven's philosophy above. If we line break something, line break at all separators at the same level as any break.

Reply:
(...unless it can trivially fit on one line)